<?xml version="1.0"?>
<launch>

  <arg name="RobotID"      default="1"/>
  <arg name="xmin" default="0.1524"/>
  <arg name="xmax" default="2.5908"/>
  <arg name="ymin" default="4.1148"/>
  <arg name="ymax" default="0.1524"/>
  <arg name="flipXY" default="False"/>

  <!-- Fake position publisher -->
  <node name="fakeRobotPoseRaster" pkg="platform_sims" type="fakeRobotPoseRaster.py" output="screen">
    <param name="xmin"     type="double"  value="$(arg xmin)"         /> # m
    <param name="xmax"     type="double"  value="$(arg xmax)"         /> # m
    <param name="Nx"       type="double"  value="64"             />
    <param name="ymin"     type="double"  value="$(arg ymin)"         /> # m
    <param name="ymax"     type="double"  value="$(arg ymax)"         /> # m
    <param name="Ny"       type="double"  value="8"              />
    <param name="stayTime" type="double"  value="0.25"           />
    <param name="flipXY"   type="bool"    value="$(arg flipXY)" /> # m
    <param name="RobotID"  type="double"  value="$(arg RobotID)" /> # m
  </node>

</launch>
